src/ODE.jl

```diff
@@ -147,6 +147,7 @@ end
 ###############################################################################
 
 include("runge_kutta.jl")
+include("adam_bashforth_methods.jl")
 
 # ODE_MS Fixed-step, fixed-order multi-step numerical method
 #   with Adams-Bashforth-Moulton coefficients
```

Comment:
Maybe you could add a note of warning that the startup is done with low order, and thus results may be of lower accuracy than expected.